main.py

```diff
@@ -59,3 +59,4 @@ def get_data(args):
 if __name__ == "__main__":
     args = parse_args(sys.argv[1:])
     create_airtable_column(args)
+if __name__ == "__main__": 
```

Comment:
hey i noticed a bug that you have an if statement that doesnt have a follow up statement and this will lead to a runtime error i'm pretty sure, could you fix that up